PHP3/insecure-hashing-non-compliant.php

```
@@ -0,0 +1,15 @@
+
+<?php
+/* 
+*  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*  SPDX-License-Identifier: Apache-2.0
+*/
+
+//{fact rule=insecure-hashing@v1.0 defects=1}
+function nonCompliant($value) {
+    // Noncompliant: Weak hash algorithm used
+    $pass = hash('md5', $value);
+    $user->setPassword($pass);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$user'. (undefined)

(UnusedLocalVariable)